src/lib/storage/index/partial_hash/partial_hash_index_impl.cpp

```diff
@@ -125,8 +125,9 @@ size_t PartialHashIndexImpl<DataType>::estimate_memory_usage() const {
   auto bytes = size_t{0u};
   bytes += sizeof(_indexed_chunk_ids);
 
-  // It is really hard to estimate the memory consumption of a HashSet. Since we are using sparse_set, the result of
-  // its .size() method should be very close to its capacity.
+  // It is not trivial to estimate the memory consumption of a hash set. We use a tsl::sparse_set, so that the size()
+  // function is closer to its actual capacity. Sparse sets/maps usually require only a few bits of overhead per entry.
+  // For rurther information about sparse sets/maps, visit https://smerity.com/articles/2015/google_sparsehash.html
```

Comment:
"size() function is closer"? "the result of size() estimates the used capacity sufficiently" or something like that?

further